sample/HttpTrigger/index.js

```diff
@@ -7,9 +7,17 @@
     }
 
     if (typeof req.query.name == 'undefined') {
-        context.done(null, "Please pass a name on the query string");
+        context.res = {
+            status: 400,
+            body: "Please pass a name on the query string"
+        };
     }
     else {
-        context.done(null, "Hello " + req.query.name);
```

Comment:
That's the error reporting mechanism. A Node pattern - if you pass an error there, the function fails and the error is logged.